vizro-mcp/README.md

```diff
@@ -0,0 +1,193 @@
+
+Alternatively, you can just ask in the chat things like:
+
+> _Create a scatter based on the iris dataset._
+
+> _Create a bar chart based on `<insert absolute file path or public URL>` data._
+
+## 🔍 Transparency and trust
+
+MCP servers are a relatively new concept, and it is important to be transparent about what the tools are capable of so you can make an informed choice as a user. Overall, the Vizro MCP server only reads data, and never writes, deletes or modifies any data on your machine.
+
+In general the most critical part of the process is the `load_and_analyze_data` tool. This tool, running on your machine, will load local or remote data into a pandas DataFrame and provide a detailed analysis of its structure and content. It only uses `pd.read_xxx`, so in general there is no need to worry about privacy or data security.
+
+The second most critical part is the `validate_model_config` tool. This tool will attempt to instantiate the Vizro model configuration and return the Python code and visualization link for valid configurations. If the configuration is valid, it will also return and attempt to open a link to a live preview of the dashboard, which will take you to [PyCafe](https://py.cafe). If you don't want to open the link, you can tell the LLM to not do so.
+
+## Available Tools (if client allows)
+
+The Vizro MCP server provides the following tools. In general you should not need to use them directly, but in special cases you could ask the LLM to call them directly to help it find its way.
+
+- `get_vizro_chart_or_dashboard_plan` - Get a structured step-by-step plan for creating either a chart or dashboard. Provides guidance on the entire creation process.
+- `get_model_JSON_schema` - Retrieves the complete JSON schema for any specified Vizro model, useful for understanding required and optional parameters.
+- `validate_model_config` - Tests Vizro model configurations by attempting to instantiate them. Returns Python code and visualization links for valid configurations.
+- `load_and_analyze_csv` - Loads a CSV file from a local path or URL into a pandas DataFrame and provides detailed analysis of its structure and content.
+- `validate_chart_code` - Validates the code created for a chart and returns feedback on its correctness.
+- `get_sample_data_info` - Provides information about sample datasets that can be used for testing and development.
+
+## Available Prompts (if client allows)
+
+- `create_starter_dashboard` - Use this prompt template to get started with Vizro dashboards.
+- `create_EDA_dashboard` - Use this prompt template to create an Exploratory Data Analysis (EDA) dashboard based on a local or remote CSV dataset
+- `create_vizro_chart` - Use this prompt template to create a Vizro styled plotly chart based on a local or remote CSV dataset
+
+## Sample data
```

Comment:
I'd make this a subsection of the section starting line 98 to make it more prominent.

Reply:
See https://github.com/mckinsey/vizro/pull/1151/commits/22292bba56864628bbe45d09fe60047a312d5c2e